ecosystem/sep-0006.md

```diff
@@ -257,6 +259,8 @@ Name | Type | Description
 `wallet_url` | string | (optional) Anchor can show this to the user when referencing the wallet involved in the withdrawal (ex. in the anchor's transaction history).
 `lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `error` fields in the response should be in this language.
 `on_change_callback` | string | (optional) A URL that the anchor should `POST` a JSON message to when the `status` property of the transaction created as a result of this request changes. The JSON message should be identical to the response format for the [/transaction](#single-historical-transaction) endpoint.
+`amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
+`country_code` | string | (optional) The [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) code of the user's current address. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
```

Comment:
❓ What happens if a country is provided that the anchor is not allowed to accept deposits from?

Reply:
In that case there is no way to help the user, the service is simply unavailable for them and the response includes a `400 Bad Request` or more specifically `422 Unprocessable Entity`.

Reply:
The anchor could provide a `choices` array too, which would help clarify whats supported. I'll add that to the example.